### PR TITLE
[Form] Added `AbstractChoiceLoader`

### LINKED ISSUE #13120: [Form] Added an AbstractChoiceLoader to simplify implementations and ha…
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#34550
| PR author(s) | @HeahDude
| Merged in    | 5.1-dev

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/symfony/symfony-docs/issues/13120

If this [one's closed](https://github.com/symfony/symfony-docs/issues/12826#issuecomment-1698641526), then this is the last one of the 5.1 milestone :smile:

### CODE REVIEW

File: reference/forms/types/options/choice_loader.rst.inc [hunk 1]
```diff
@@ -26,6 +26,22 @@ This will cause the call of ``StaticClass::getConstants()`` to not happen if the
...
 
+If the ``CallbackChoiceLoader`` doesn't fit your needs and you want to create
+your own loader, you can either create a loader by implementing the
+:class:`Symfony\\Component\\Form\\ChoiceList\\Loader\\ChoiceLoaderInterface`
+or by extending the
+:class:`Symfony\\Component\\Form\\ChoiceList\\Loader\\AbstractChoiceLoader`.
+This abstract class allows to save some boilerplate by implementing some
+of the :class:`Symfony\\Component\\Form\\ChoiceList\\Loader\\ChoiceLoaderInterface`
+methods. Thus, you'll only have to implement the
+:class:`Symfony\\Component\\Form\\ChoiceList\\Loader\\AbstractChoiceLoader::loadChoices`
```
Comment (HeahDude) [inline reference/forms/types/options/choice_loader.rst.inc]:
```suggestion
:method:`Symfony\\Component\\Form\\ChoiceList\\Loader\\AbstractChoiceLoader::loadChoices`
```

**Review by HeahDude — COMMENTED** [review]
Thanks for documenting this :)

File: reference/forms/types/options/choice_loader.rst.inc (hunk 1 above)
Comment (alexandre-daubois) [reply reference/forms/types/options/choice_loader.rst.inc]: Fixed, thanks!

**Comment by javiereguiluz** [comment]
Alex, thanks for contributing this missing doc!

**Pull request merged**